spec/features/user_gets_sitemap_spec.rb

```diff
@@ -0,0 +1,18 @@
+feature "User gets sitemap" do
+
+  let!(:user){ Fabricate(:user) }
+  let!(:foo_post){ Fabricate(:post, author: user, title: "Foo", body: "Bar") }
+  let!(:user2){ Fabricate(:user) }
+  let!(:bar_post){ Fabricate(:post, author: user2, title: "Foo", body: "Bar") }
+
+  scenario "GET Full Sitemap" do
+    visit sitemaps_path
+    expect(page.source).to include('<url>')
+    expect(page.source).to include('<changefreq>monthly</changefreq>')
+    expect(page.source).to include('users.1')
```

Comment:
These aren't the routes that you should be looking for.

You should be looking for /users/1 or /users/1/posts, etc.  Also, these hardcoded ids won't work if this isn't the first test run.

You should use the route helpers.